spec/rails_helper.rb

```diff
@@ -92,16 +92,6 @@
   config.filter_rails_from_backtrace!
   # arbitrary gems may also be filtered via:
   # config.filter_gems_from_backtrace("gem name")
-
-  config.before(:suite) do
-    Rails.application.load_seed
```

Comment:
Took me too long to find this so I moved it, and the feature flag setup below, into their own support files. Plus decoupled from the actual seeds file and instead call the specific seeders we want.